index.bs

```diff
@@ -719,36 +990,45 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 		<h4 method for="Clipboard">writeText(|data|)</h4>
 		The {{Clipboard/writeText(data)}} method must run these steps:
 
-			1. Let |p| be a new [=Promise=].
+			1. Let |realm| be [=this=]'s [=relevant realm=].
+
+			1. Let |p| be [=a new promise=] in |realm|.
 
 			1. Run the following steps [=in parallel=]:
 
-				1. Let |r| be the result of running [=check clipboard write permission=] [=in parallel=]
+				1. Let |r| be the result of running [=check clipboard write permission=].
+
+					Note: Clipboard permission is not supported on Safari. However, the writeText() method must be called inside
```

Comment:
hm that hasn't been merged yet, and since we have this in Chromium, I'm not sure if we want to make that change. Let's first discuss with the Editing WG and then decide what needs to be done here.

Reply:
Added.